hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OMThroughputBenchmark.java

```diff
@@ -0,0 +1,483 @@
+
+    @Override
+    public void runTask(long counter) throws Exception {
+      OmKeyArgs keyArgs = new OmKeyArgs.Builder()
+          .setVolumeName(volumeName)
+          .setBucketName(bucketName)
+          .setKeyName(generateObjectName(counter))
+          .setRefreshPipeline(true)
+          .build();
+      getTimer().time(() -> {
+        ozoneManagerClient.lookupKey(keyArgs);
+        return null;
+      });
+    }
+  }
+
+  private class RenameKeyTask extends DeleteKeyTask {
+
+    @Override
+    public void runTask(long counter) throws Exception {
+      OmKeyArgs keyArgs = new OmKeyArgs.Builder()
+          .setVolumeName(volumeName)
+          .setBucketName(bucketName)
+          .setKeyName(generateObjectName(counter))
+          .build();
+      getTimer().time(() -> {
+        ozoneManagerClient.renameKey(keyArgs,
+            generateNewObjectName(generateObjectName(counter)));
+        return null;
+      });
+
+    }
+  }
+
+  private class ListKeyTask extends DeleteKeyTask {
+
+    @Override
+    public void runTask(long counter) throws Exception {
+      getTimer().time(() -> {
+        ozoneManagerClient.listKeys(
+            volumeName, bucketName, null, getPrefix(), 10);
+        return null;
+      });
+    }
+  }
+
+  private class CreateFileTask extends ApiTask {
+
+    @Override
+    void runTask(long counter) throws Exception {
+      UserGroupInformation ugi = UserGroupInformation.getCurrentUser();
+      OmKeyArgs keyArgs = new Builder()
+          .setBucketName(bucketName)
+          .setVolumeName(volumeName)
+          .setReplicationConfig(new RatisReplicationConfig(factor))
+          .setKeyName(generateObjectName(counter))
+          .setLocationInfoList(new ArrayList<>())
+          .setAcls(OzoneAclUtil.getAclList(ugi.getUserName(),
+              ugi.getGroupNames(), ALL, ALL))
+          .build();
+
+      getTimer().time(() -> {
+        OpenKeySession openKeySession = ozoneManagerClient.createFile(keyArgs,
+            true, true);
+
+        ozoneManagerClient.commitKey(keyArgs, openKeySession.getId());
+
+        return null;
+      });
+
+    }
+
+    @Override
+    public void prepare() throws Exception {
+      ensureVolumeAndBucketExist(ozoneClient, volumeName, bucketName);
+      setTimer();
+    }
+  }
+
+  private class DeleteFileTask extends ApiTask {
+
+    @Override
+    void runTask(long counter) throws Exception {
+      UserGroupInformation ugi = UserGroupInformation.getCurrentUser();
+      OmKeyArgs keyArgs = new Builder()
+          .setBucketName(bucketName)
+          .setVolumeName(volumeName)
+          .setReplicationConfig(new RatisReplicationConfig(factor))
+          .setKeyName(generateObjectName(counter))
+          .setLocationInfoList(new ArrayList<>())
+          .setAcls(OzoneAclUtil.getAclList(ugi.getUserName(),
+              ugi.getGroupNames(), ALL, ALL))
+          .build();
+
+      getTimer().time(() -> {
+        ozoneManagerClient.getFileStatus(keyArgs);
```

Comment:
is getFileStatus() necessary? It would skew the throughput of delete

Reply:
> 

You're right. Originally I followed the process of delete file requests, which have this step first. This could be removed.